backend/src/main/java/com/pasdaven/backend/controller/UserAccountController.java

```diff
@@ -66,4 +66,16 @@ public ResponseEntity<UserAccountEntity> getUserAccountById(@PathVariable Intege
         UserAccountEntity userAccount = userAccountService.getUserAccountByEmail(user.getUserAccount().getEmail());
         return new ResponseEntity<>(userAccount, HttpStatus.OK);
     }
+
+    @GetMapping("/id/")
```

Comment:
> I think it is a bit strange to use "id" as the URL of the API, because its access method is through token, and the returned data is not pure id

Change URL to "tokenId"